fe/fe-core/src/main/java/org/apache/doris/nereids/rules/rewrite/MergeAggregate.java

```diff
@@ -75,7 +74,10 @@ public List<Rule> buildRules() {
      */
     private Plan mergeTwoAggregate(LogicalAggregate<LogicalAggregate<Plan>> outerAgg) {
         LogicalAggregate<Plan> innerAgg = outerAgg.child();
-
+        Map<ExprId, AggregateFunction> innerAggExprIdToAggFunc = innerAgg.getOutputExpressions().stream()
+                .filter(expr -> (expr instanceof Alias) && (expr.child(0) instanceof AggregateFunction))
+                .collect(Collectors.toMap(NamedExpression::getExprId, value -> (AggregateFunction) value.child(0),
+                        (existValue, newValue) -> existValue));
```

Comment:
Don't repeat yourself. This code repeat 3 times, add a private method to compute it?